src/bridge_stream.rs

```diff
@@ -330,13 +337,16 @@ mod tests {
 
         let stream1 = futures::stream::iter(vec![item1]);
         let stream2 = futures::stream::iter(vec![item2]);
+        println!("Made streams");
```

Comment:
Switch to using log and instantiate env_logger like in the other tests.

Reply:
Should be GTG